core/modules/taxonomy/taxonomy.theme.inc

```diff
@@ -25,7 +25,9 @@ function theme_taxonomy_overview_vocabularies($variables) {
 
       $row = array();
       $row[] = backdrop_render($vocabulary['name']);
-      $row[] = backdrop_render($vocabulary['items']);
+      // Align these cells to the right. The align attribute of <td> elements is
+      // not supported in HTML5, so we use CSS instead.
+      $row[] = array('data' => backdrop_render($vocabulary['items']), 'style' => 'text-align: right');
```

Comment:
I dislike in-lining style. Is there not a stylesheet we can add this to somewhere?